### PR TITLE
General 0-Order VFA Recovery Unit

### PR DESCRIPTION
**Pull request by adam-a-a**
## Summary/Motivation:
Adding a zero-order volatile fatty acids (VFA) recovery unit for the AMO effort.

## Changes proposed in this PR:
- add VFA recovery unit and test
- add yaml
-  add new component to list for ffCOD, representative of VFAs

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: watertap/data/techno_economic/component_list.yaml [hunk 1]
```diff
@@ -64,6 +64,8 @@ nitrite:
   constituent_longform: Nitrite
 nitrogen:
   constituent_longform: Nitrogen
+nonbiodegradable_cod:
+  constituent_longform: Nonbiodegradable Chemical Oxygen Demand (ffCOD)
```
Comment (bknueven) [inline watertap/data/techno_economic/component_list.yaml:68]: Should this (and the `nonvolatile_toc` below) have names without acronyms? E.g.
```suggestion
nonbiodegradable_chemical_oxygen_demand:
  constituent_longform: Nonbiodegradable Chemical Oxygen Demand (ffCOD)
```

**Review by bknueven — APPROVED** [review]
I have a naming question. But this is okay as-is.

File: watertap/data/techno_economic/component_list.yaml (hunk 1 above)
Comment (andrewlee94) [reply watertap/data/techno_economic/component_list.yaml:68]: Generally we have used shorter names as the key, and expanded on them in the `constituent_longform`.

For my part, what does the `ff` in `ffCOD` mean (i.e. just making sure it isn't a typo or issue with importing the csv files)?

EDIT: I see a more detailed longform in a later file - we should probably make these the same.

**Review by andrewlee94 — APPROVED** [review]
Looks good to me.

File: watertap/data/techno_economic/component_list.yaml (hunk 1 above)
Comment (adam-a-a) [reply watertap/data/techno_economic/component_list.yaml:68]: ffCOD stands for flocculated and filtered chemical oxygen demand. In the case of this particular unit, ffCOD is representative of volatile fatty acids (VFAs), hence the more elaborate longform presented in the unit's yaml file. However, the general longform that isn't unit specific should not describe this component as "representative of volatile fatty acids", which is why I wrote it this way here.

Comment (andrewlee94) [reply watertap/data/techno_economic/component_list.yaml:68]: As an observation then, we want to be really careful with things like this. What happens if we have two unit operations in a process which use different definitions for the same component (i.e. one unit uses the more elaborate definition, and one uses a more general definition)?

**Pull request merged**